src/shared/i18next.ts

```diff
@@ -63,7 +63,7 @@ export const languages = [
   { resource: sv, code: "sv", name: "Svenska" },
   { resource: vi, code: "vi", name: "Tiếng Việt" },
   { resource: zh, code: "zh", name: "中文 (简体)" },
-  { resource: zh_Hant, code: "zh_Hant", name: "中文 (繁體)" },
+  { resource: zh_Hant, code: "zh-TW", name: "中文 (繁體)" },
```

Comment:
Must use an underscore.

Reply:
Could you elaborate more why we must use an underscore here?
I have tested on my machine. It seems that it's not OK to use underscores for both zh_TW and  pt_BR.
Lemmy (without a login) will not pick the correct language based on browser settings.

Reply:
My browser setting
`navigator.languages:   [ "zh-TW", "en-US", "en" ]`

Reply:
Ah... I was under the impression it needed one, coulda swore I tested that with pt-BR. In that case I'll trust your judgement here, thx!